src/AsyncSessionMiddleware.php

```diff
@@ -2,42 +2,50 @@
 
 namespace Interop\Session\Middleware\Async;
 
-use Psr\Http\Message\{ServerRequestInterface as Request, ResponseInterface as Response};
-use Interop\Session\Manager\{SessionManagerInterface as SessionManager};
+use Psr\Http\Message\{
+    ServerRequestInterface as Request, ResponseInterface as Response
+};
+use Interop\Session\Manager\{
+    SessionManagerInterface as SessionManager
+};
 use Interop\Session\Utils\ArraySession\SavableSession as Session;
 use Zend\Stratigility\MiddlewareInterface;
 use Interop\Session\SessionInterface;
 
-class AsyncSessionMiddleware implements MiddlewareInterface{
+class AsyncSessionMiddleware implements MiddlewareInterface
```

Comment:
MiddlewareInterface is deprecated in Stratigility 1.3.
See: https://github.com/zendframework/zend-stratigility/blob/master/doc/book/middleware.md